sap/cli/gcts.py

```diff
@@ -320,13 +334,14 @@ def pull(connection, args):
     console = sap.cli.core.get_console()
 
     try:
+        repo = get_repository(connection, args.package)
         with sap.cli.helpers.ConsoleHeartBeat(console, args.heartbeat):
-            response = sap.rest.gcts.simple.pull(connection, name=args.package)
+            response = sap.rest.gcts.simple.pull(connection, repo=repo)
```

Comment:
I am not sure we need to use the simplified API here. Just use call repo.pull directly.